DABI.js

```diff
@@ -0,0 +1,321 @@
+{
+    "translatorID": "5cf8bb21-e350-444f-b9b4-f46d9fab7827",
+    "label": "DABI",
+    "creator": "Jens Mittelbach",
+    "target": "^https?://dabi\\.ib\\.hu-berlin\\.de/",
+    "minVersion": "1.0",
+    "maxVersion": "",
+    "priority": 100,
+    "inRepository": true,
+    "translatorType": 4,
+    "browserSupport": "gcsibv",
+    "lastUpdated": "2016-01-08 11:43:21"
+}
+
```

Comment:
Add here a license block, which should like https://github.com/zuphilip/translators/wiki/Common-code-blocks-for-translators#licence-block

Reply:
done